src/config/jest.config.js

```diff
@@ -15,8 +15,9 @@ const ignores = [
 const jestConfig = {
   roots: [fromRoot('src')],
   testEnvironment: ifAnyDep(['webpack', 'rollup', 'react'], 'jsdom', 'node'),
-  collectCoverageFrom: ['src/**/*.js'],
-  testMatch: ['**/__tests__/**/*.js'],
+  moduleFileExtensions: ['js', 'jsx', 'json', 'ts', 'tsx'],
+  collectCoverageFrom: ['src/**/*.js', 'src/**/*.js'],
+  testRegex: '/__tests__/.*\\.(ts|tsx|js)$',
```

Comment:
Could this be:

```
testMatch: ['**/__tests__/**/*.+(js|jsx|ts|tsx)'],
```

Reply:
Interesting, I will test this. For some reason I thought `testRegex` had to be used for regex, and testMatch only accepted list of type. But I'll test and verify.